summit/develop/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "3.6"
+
+services:
+
+  develop-env:
+    image: ts-dockerhub.lsst.org/develop-env:${CYCLE}${rev}
+    container_name: develop-env
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=summit
+      - OSPL_INFOFILE=/tmp/ospl-info-daemon.log
```

Comment:
You should use something other than `daemon` here and the next line since this isn't a daemon container.